osrf/vrx
# Gazebo Garden and ROS Humble

Requirements:
  * Ubuntu 22.04
  * ROS Humble ([Installation instructions](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html)). I installed ` ros-humble-desktop` from debs.
  * ROS Garden ([Installation instructions](https://gazebosim.org/docs/garden/install_ubuntu_src)). I installed from source, as Garden hasn't been released yet.

How to test it?

1. Compile VRX following [the updated instructions](https://github.com/osrf/vrx/wiki/2023-installation#installation-from-source). 
2. Launch the simulation following the [updated instructions](https://github.com/osrf/vrx/wiki/2023-running_vrx).

You should see the usual Sydney Regatta with the WAM-V.

Big thanks to @srmainwaring for helping with the port.
